### PR TITLE
Excellence

### PR DESCRIPTION
**Pull request by bakerkretzmar**
This PR brings Ziggy closer to _excellence_:

- add `.editorconfig`
- update `.gitattributes` and `.gitgnore`
- un-track `package-lock.json`
- update `package.json` and `composer.json`
- update `contributing.md`
- update the issue template
- update `readme.md`
  - grammar and wording
  - consistent formatting
  - badges
  - Security section

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,15 +2,25 @@
...
+        "url": "https://github.com/tightenco/ziggy.git"
     },
-    "author": "Daniel Coulbourne",
+    "authors": [
+        {
+            "name": "Daniel Coulbourne",
+            "email": "daniel@tighten.co"
+        },
+        {
+            "name": "Matt Stauffer",
```
Comment (mattstauffer) [inline package.json:20]: I don't need to be listed on author on a project where I did very little! :) But I'm fine for Jake and Jacob to be added, if y'all write much code.

**Comment by ankurk91** [comment]
@bakerkretzmar 
What about using Github actions for testing to make it more awesome.

File: README.md [hunk 2]
```diff
@@ -229,92 +230,99 @@ Route::get('/login', function () {
...
+## Contributing
 
 To get started contributing to Ziggy, check out [the contribution guide](CONTRIBUTING.md).
 
+## Credits
+
 - [Daniel Coulbourne](https://twitter.com/DCoulbourne)
 - [Jake Bathman](https://twitter.com/jakebathman)
 - [Matt Stauffer](https://twitter.com/stauffermatt)
+- [All contributors](https://github.com/tightenco/ziggy/contributors)
```
Comment (jakebathman) [inline README.md:318]:
```suggestion
- [Jacob Baker-Kretzmar](https://twitter.com/bakerkretzmar)
- [All contributors](https://github.com/tightenco/ziggy/contributors)
```

Comment (bakerkretzmar) [reply README.md:339]: 🙏

**Pull request merged**